adala/skills/collection/label_studio.py

```diff
@@ -62,10 +72,17 @@ async def aapply(
     ) -> InternalDataFrame:
 
         with json_schema_to_pydantic(self.field_schema) as ResponseModel:
-            return await runtime.batch_to_batch(
+            output = await runtime.batch_to_batch(
                 input,
                 input_template=self.input_template,
                 output_template="",
                 instructions_template=self.instructions,
                 response_model=ResponseModel,
             )
+            ner_tag = self.has_ner_tag()
+            if ner_tag:
```

Comment:
why do we need to handle ner tag with a special case?

Reply:
to match true indices in text as a post processing step